code-challenges.js

```diff
@@ -6,7 +6,12 @@
 var people = [{name: "ford prefect", occupation: "hitchhiker"}, {name: "zaphod beeblebrox", occupation: "president of the galaxy,"}, {name: "arthur dent", occupation: "radio employee"}]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is the president of the galaxy." "Arthus Dent is a radio employee."
 
+const sentenceGenerator = (array) => {
+    let sentences = array.map(value => `${value.name} is a ${value.occupation}.`)
+    return sentences
+}
```

Comment:
great work, this is really concise! just think of how you might handle the return saying "is the president" instead of "is a"